launchpad/interaction/snippets.sh

```diff
@@ -1,7 +1,7 @@
 OWNER_PEM_PATH="elrondOwner.pem"
```

Comment:
Not really relevant, but I would also change it here as well.

Reply:
done.